src/hash/blake/mod.rs

```diff
@@ -91,13 +108,37 @@ impl ElementHasher for Blake3_256 {
     }
 }
 
+impl Blake3_256 {
+    /// Returns a hash of the provided sequence of bytes.
+    #[inline(always)]
```

Comment:
The `inline` will most likely be performed by the compiler, regardless. It might be best to leave to him to decide when to inline or not, unless we have a strong reason to add this directive